website/static/js/fangorn.js

```diff
@@ -1348,7 +1348,7 @@ function gotoFileEvent (item, toUrl) {
     }
 
     if (COMMAND_KEYS.indexOf(tb.pressedKey) !== -1) {
-        window.open(fileurl, '_blank');
+        window.open(fileurl, item.data.materialized);
```

Comment:
If product agrees with this behavior, you'll want to check the fangornconfigs for bitbucket, figshare, github, dataverse since some of them overwrite this behavior. I know for sure github does.

Reply:
Yes, when it's accepted I will take the time to test similar fix for other addons.

Reply:
Cool, I moved it to demo then 👍